ui/ozone/platform/starboard/test/platform_window_starboard_unittest.cc

```diff
@@ -25,36 +25,77 @@ namespace ui {
 namespace {
 class PlatformWindowStarboardTest : public testing::Test {
  public:
-  PlatformWindowStarboardTest() : sb_window_(&delegate_, gfx::Rect(0, 0)) {}
+  PlatformWindowStarboardTest() {}
 
   ~PlatformWindowStarboardTest() = default;
+};
+
+// Helper function to create a basic event.
+ui::Event* CreateTestEvent() {
+  return new ui::MouseEvent(ui::ET_MOUSE_PRESSED, gfx::Point(0, 0),
+                            gfx::Point(0, 0), base::TimeTicks::Now(), 0, 0);
+}
 
-  PlatformWindowStarboard* window() { return &sb_window_; }
-  TestPlatformDelegate delegate() { return delegate_; }
+TEST_F(PlatformWindowStarboardTest, CreateAndDestroy) {
+  TestPlatformWindowDelegate delegate;
+  gfx::Rect bounds(100, 200, 300, 400);
```

Comment:
nit: `constexpr gfx::Rect kBounds(100,...);` here and elsewhere (`gfx::Rect` has a constexpr constructor).